CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+- Add: implement `encode` function for summary type metrics. 
```

Comment:
```suggestion
- Add: Implement `encode` function for summary type metrics. 
```

Reply:
@mxinden Thanks for such a quick response - thanks for pointing out those tests, I had missed them, and have added a simple summary test as well.